components/button/button-group.tsx

```diff
@@ -25,8 +26,11 @@ const ButtonGroup: React.SFC<ButtonGroupProps> = props => (
           break;
         case 'small':
           sizeCls = 'sm';
-        default:
+        case 'default':
+        case undefined:
           break;
+        default:
+          throw new UnreachableException(size);
```

Comment:
`throw` could too aggressive for a library, maybe `console.error`?

Reply:
use `warning`

Reply:
ping @fa93hws